pil-analyzer/tests/types.rs

```diff
@@ -516,3 +516,13 @@ fn enum_too_many_fields() {
     ";
     type_check(input, &[]);
 }
+
```

Comment:
Can you add another test that checks types for empty match arms, empty if/else blocks and just empty blocks inside fuctions? - or at least blocks with empty expression part.